cmd/cmd.go

```diff
@@ -83,12 +99,27 @@ var dropCmd = &cobra.Command{
 		if err != nil {
 			return err
 		}
-		pg, err := db.NewPostgreSQL(u, postgresSchema, nil)
-		if err != nil {
+		uri := os.Getenv("DATABASE_URL")
+		if strings.HasPrefix(uri, "mongodb://") {
+			mdb, err := db.NewMongoDB(uri)
+			if err != nil {
+				return err
+			}
+			err = mdb.DropCollection()
+			if err != nil {
+				return err
+			}
 			return err
+		} else if strings.HasPrefix(uri, "postgres://") || strings.HasPrefix(uri, "postgresql://") {
+			pg, err := db.NewPostgreSQL(u, postgresSchema, nil)
+			if err != nil {
+				return err
+			}
+			defer pg.Close()
+			return pg.DropTable()
+		} else {
+			return fmt.Errorf("A URL não contém 'mongodb' nem 'postgres'")
```

Comment:
Corrigindo o erro que a CI acusa

```suggestion
			return fmt.Errorf("a URL não contém 'mongodb' nem 'postgres'")
```